.github/workflows/.build.yml

```diff
@@ -63,14 +63,13 @@ jobs:
                     includes.push({
                       runnner: platform.startsWith('linux/arm') ? 'ubuntu-24.04-arm' : 'ubuntu-24.04',
                       distro: distro,
-                      platform: platform
+                      platform: platform,
+                      verify: platform.startsWith('linux/386') || platform.startsWith('linux/amd64') || platform.startsWith('linux/arm64')
```

Comment:
Why not verify all `linux/`?

Reply:
It would segfault through emulation atm, I can add a TODO comment.